app/src/main/java/com/duckduckgo/app/settings/db/SettingsDataStore.kt

```diff
@@ -85,6 +86,10 @@ class SettingsSharedPreferences @Inject constructor(private val context: Context
         }
         set(appIcon) = preferences.edit(commit = true) { putString(KEY_APP_ICON, appIcon.componentName) }
 
+    override var appIconChanged: Boolean
```

Comment:
Could we add a comment here explaining this flag? i just know otherwise i'm going to forget it's a temporary flag just across the app restart

Reply:
Yeah that make sense, I've added a comment explaining it